### PR TITLE
[Feature #162727567]: Estimate read time of articles

### PR DESCRIPTION
**Pull request by rajeman**
#### What does this PR do?
- it estimates read time for created articles
#### Description of Task to be completed?
- [x] create a helper for estimating read time
- [x] setup a migration file that adds a `readTime` column to Article model

#### How should this be manually tested?
- create an article at `POST /articles` and the estimated read time will be returned in the response object

#### Any background context you want to provide?
- Currently our articles do not contain images. Therefore this readTime estimator works with only text. It will be upgraded once image markdown format is determined

#### What are the relevant pivotal tracker stories?
[#162727567](https://www.pivotaltracker.com/n/projects/2229853/stories/162727567)

### CODE REVIEW

File: helpers/getReadTime.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default (text) => {
+  const wordCount = text.match(/\S+/g).length;
+  const readTime = Math.floor(wordCount / 275);
+  return readTime || 1;
```
Comment (oreoluwade) [inline helpers/getReadTime.js:4]: Does `1` here account for when the number of words is very little?

If yes, I think you should write one extra test spec showing that when the article body is sufficiently long, the result of `readTime` is something other than 1.

A lorem ipsum-ish extensive body of text could help here.

Comment (rajeman) [reply helpers/getReadTime.js:4]: I have done that

File: test/010-helpers.test.js [hunk 2]
```diff
@@ -28,4 +28,18 @@ describe('HELPER TEST SUITE', () => {
...
+
+  describe('getReadTime test suite', () => {
+    it('Should return 1 for texts less than 550 words', (done) => {
+      const readTime = getReadTime('There is fire on the mountain');
+      expect(readTime).to.equal(1);
+      done();
+    });
+
+    it('Should get the read time of texts with over 550 words', (done) => {
+      const readTime = getReadTime('word '.repeat(1000));
```
Comment (oreoluwade) [inline test/010-helpers.test.js:40]: 💯

**Review by oreoluwade — APPROVED** [review]

**Comment by oreoluwade** [comment]
I saw the background context. Thanks for pointing that out. We'd be looking to update as soon as we get images in.

**Pull request merged**